### PR TITLE
Added support for carbon parameters to grid_params.py

### LINKED ISSUE #30: Add support for carbon variables
Add support for carbon variables

### PR DESCRIPTION
**Pull request by tbohn**
This is branched off of PR #40.  After PR #40 is merged, I will rebase this.

Closes #30.

### CODE REVIEW

**Comment by jhamman** [comment]
I'm going to wait on #40 before reviewing this one.

**Comment by tbohn** [comment]
Strange, I rebased relative to the previous PR, but the diff still shows all of the changes of both PRs combined...

**Comment by tbohn** [comment]
Wait, never mind, the diff shows all the changes of both PRs because the first PR hasn't been merged yet.

**Comment by tbohn** [comment]
When the previous PR is merged, I'll rebase this against develop.

**Comment by jhamman** [comment]
okay, rebase...

**Comment by tbohn** [comment]
OK, rebased.  Waiting for travis build.

File: tonic/models/vic/grid_params.py [hunk 1]
```diff
@@ -1357,6 +1426,11 @@ def veg_class(vegl_file, c=Cols(veglib_fcan=False)):
                 for col in np.arange(0, col_desc):
                     data[row - 1][:] = words[:col_desc]
                     data[row - 1].append(sep.join(words[col_desc:]))
+                if veglib_photo:
+                    if data[row - 1][c.veglib['lib_Ctype']] == 'C3':
+                        data[row - 1][c.veglib['lib_Ctype']] = 0
+                    elif data[row - 1][c.veglib['lib_Ctype']] == 'C4':
+                        data[row - 1][c.veglib['lib_Ctype']] = 1
```
Comment (jhamman) [inline tonic/models/vic/grid_params.py]: I don't get this. Let's think about how to continue using the string representation.  It is certainly easier to ready :wink:

Comment (tbohn) [inline tonic/models/vic/grid_params.py]: So, what do you want me to do about this, given that you OK'ed the PR having VIC read either strings or ints?

Comment (jhamman) [inline tonic/models/vic/grid_params.py]: This if fine for now.  I'll play around with this later on...

**Pull request merged**